user-list-prac-6/src/components/ProfileList/profileList.css

```diff
@@ -0,0 +1,143 @@
+.user-list {
+  height: 590px;
+}
+th {
+  color: #4d4d55;
+  font-weight: bold;
+  font-size: 18px;
+  padding: 5px 50px 14px 50px;
+  text-align: center;
+}
+td {
+  padding: 20px 50px;
+}
+
+td:first-child {
```

Comment:
remove this and do something like
     `<th style="width: 30%">Col 1</th>`
     `<th style="width: 20%">Col 2</th>`
      `<th style="width: 10%">Col 3</th>`